src/Jackett.Server/Controllers/ServerConfigurationController.cs

```diff
@@ -92,6 +92,8 @@ public IActionResult UpdateConfig([FromBody] Common.Models.DTO.ServerConfig conf
                     throw new Exception("The Base Path Override must start with a /");
```

Comment:
```suggestion
                    throw new Exception("The Base Path Override must start with a <b>/</b>");
```
just for consistency